wgsl/index.bs

```diff
@@ -2311,7 +2464,7 @@ This includes the [=store type=] of a workgroup variable.
 <div class='syntax' noexport='true'>
   <dfn for=syntax>array_type_specifier</dfn> :
 
-    | <a for=syntax_kw lt=array>`'array'`</a> <a for=syntax_sym lt=less_than>`'<'`</a> [=syntax/type_specifier=] ( <a for=syntax_sym lt=comma>`','`</a> [=syntax/element_count_expression=] ) ? <a for=syntax_sym lt=greater_than>`'>'`</a>
+    | <a for=syntax_kw lt=array>`'array'`</a> <span class=hidden>_disambiguate_template</span> <a for=syntax_sym lt=_template_args_start>_template_args_start</a> [=syntax/type_specifier=] ( <a for=syntax_sym lt=comma>`','`</a> [=syntax/element_count_expression=] ) ? <a for=syntax_sym lt=_template_args_end>_template_args_end</a>
```

Comment:
Here's the first instance of inserting a _disambiguate_template token.
It's done with a `<span....` with class=hidden

extract-grammar.py has been taught to recognize that and generate the _disambiguate_template symbol name.